src/utils/generatePaymentLink.ts

```diff
@@ -1,11 +1,10 @@
 import { PaymentDetails } from "../../types";
 import env from "../../Config"
-// import * as mnid from '@types/mnid'
-
+import * as mnid from 'mnid';
 
 export function generatePaymentLink({address, recipient, amount, reason, category, vendorData}:PaymentDetails) : string {
     const codeObj = {
-        m: recipient,
+        m: '',
```

Comment:
whats that? where is recipient used?

Reply:
we need mnid in this variable for the payment link. For now recipient isn't used anywhere do we need to incorporate new variable which store 'recipient'?

Reply:
if recipient is not used then remove it

Reply:
Got it!